test/unit/tech/flash.test.js

```diff
@@ -70,6 +70,7 @@ test('dispose removes the object element even before ready fires', function() {
   mockFlash.off = noop;
   mockFlash.trigger = noop;
   mockFlash.el_ = {};
+  mockFlash.textTracks = () => ([]);
```

Comment:
I think this is what was causing the issues in safari that we're seeing in master and elsewhere.